Starting Container

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'

Traceback (most recent call last):

  File "/app/bot.py", line 7, in <module>

    from aiogram.enums import ParseMode

ModuleNotFoundError: No module named 'aiogram.enums'
